curvefs/src/metaserver/inode_storage.cpp

```diff
@@ -380,22 +388,23 @@ size_t InodeStorage::Size() {
 }
 
 MetaStatusCode InodeStorage::Clear() {
-    ReadLockGuard w(rwLock_);
+    WriteLockGuard w(rwLock_);
     Status s = kvStorage_->HClear(table4inode_);
     if (!s.ok()) {
-        LOG(ERROR) << "InodeStorage HClear() failed";
+        LOG(ERROR) << "InodeStorage HClear() failed, error: " << s.ToString();
         return MetaStatusCode::STORAGE_INTERNAL_ERROR;
     }
     s = kvStorage_->SClear(table4s3chunkinfo_);
     if (!s.ok()) {
-        LOG(ERROR) << "InodeStorage SClear() failed";
+        LOG(ERROR) << "InodeStorage SClear() failed, error: " << s.ToString();
         return MetaStatusCode::STORAGE_INTERNAL_ERROR;
     }
 
```

Comment:
shoudle we clean `table4volumeExtent` table?

Reply:
fix